### DISCUSSION TITLE
Flutter Firebase Phone Authentication is working with test Number but not working with other number?

### DISCUSSION THREAD

**Discussion by vijaypareek13** [firebase/flutterfire#5492] [Q&A]
I have tried adding release and debug Sha1 and sha256 but all in vain.

**Comment by Pr47h4m** [comment]
Hi 
I've recently implemented phone authentication in my project and I also faced same issue
be sure you have enabled "Android Device Verification" api from google cloud platform 
you can do it by following steps below:
* open [Google Cloud Console](https://console.cloud.google.com)
* navigate to API & Services from navigation menu
* search for "Android Device Verification" API and enable it 
then try phone authentication once again it should work (considering you have done flutter and firebase setup correctly)
